glean-core/android/src/main/java/mozilla/telemetry/glean/private/StringListMetricType.kt

```diff
@@ -153,6 +153,6 @@ class StringListMetricType(
         } catch (e: org.json.JSONException) {
             throw NullPointerException()
         }
-        return jsonRes.toList<String>()
+        return jsonRes.toList()
```

Comment:
We're really only using `toList` (and `asSequence`, used by it): let's not import other code which we won't need.